scripts/pre_commit_linter.py

```diff
@@ -292,6 +292,9 @@
     os.path.join(_PARENT_DIR, 'oppia_tools', 'pylint-quotes-0.1.9'),
     os.path.join(_PARENT_DIR, 'oppia_tools', 'selenium-2.53.2'),
     os.path.join(_PARENT_DIR, 'oppia_tools', 'PIL-1.1.7'),
+    os.path.join(_PARENT_DIR, 'oppia_tools', 'GitPython-2.1.11'),
```

Comment:
Place `smmap` first then `gitdb` and then `GitPython` (in the order of the dependence relation).

Reply:
Not done?

Reply:
So sorry -- done now